plugins/modules/filesystem.py

```diff
@@ -79,6 +79,14 @@
     description:
       - List of options to be passed to C(mkfs) command.
     type: str
+  uuid:
+    description:
+      - Set existing filesystem's UUID to the given value.
```

Comment:
I think this should also work on FS creation.

Reply:
Good idea, I'm working on the necessary changes.